src/api/rawModelTypes.js

```diff
@@ -80,13 +53,6 @@ export const transformFetchedMessage = <M: Message>(
     userId: message.sender_id,
     realm: identity.realm,
   }),
-  reactions: message.reactions.map(reaction => {
-    const { user, ...restReaction } = reaction;
-    return {
-      ...restReaction,
-      user_id: user.id,
```

Comment:
Oof. Yeah, what we should have done long ago would be to make this say `user.id ?? restReaction.user_id` (or perhaps, equivalently, `user.id ?? user_id` after saying `const { user_id, user, ...restReaction }` above… or perhaps better yet, write the fallback the other way around with modern first, like `user_id ?? user.id`).

Then it would have already been forward-compatible with an eventual server version that dropped the old `user` property, without really being materially more complicated.